spec/support/fake_analytics_spec.rb

```diff
@@ -0,0 +1,350 @@
+require 'rails_helper'
+
+RSpec.describe FakeAnalytics do
+  subject { described_class.new }
```

Comment:
WDYT of naming this subject and using that name in the specs?
```suggestion
  subject(:fake_analytics) { described_class.new }
```